api/v1/subschema/sauce.schema.json

```
@@ -73,6 +73,13 @@
             { "const": "team", "title": "Only accessible to people under the same root account as you." },
             { "const": "private", "title": "Only you (the owner) will be able to view assets and test results page." }
           ]
+        },
+        "launchOrder": {
+          "description": "Launch jobs based on the failure rate. Jobs with the highest failure rate launch first.",
+          "type": "string",
+          "oneOf": [
+            { "const": "fail rate", "title": "Launch job based on the failure rate"}
```

Need to add a bit more detail, since it's not clear what you mean by "based on failure rate". Lowest first? Highest first? What's the order?

```suggestion
            { "const": "fail rate", "title": "Suites that historically have the highest failure rate start first."}
```